backend/item/repo.go

```diff
@@ -0,0 +1,66 @@
+package item
+
+import (
+	"github.com/SAKA-club/todo/backend/gen/models"
+	"github.com/jmoiron/sqlx"
+	"github.com/rs/zerolog/log"
+	"time"
+)
+
+type repo struct {
+	db *sqlx.DB
+}
+
+func NewRepo(db *sqlx.DB) *repo {
+	return &repo{db: db}
+
+}
+
+func (r repo) GetAll() ([]*models.Item, error) {
+	var items []*models.Item
+	if err := r.db.Select(&items, `SELECT id, title, priority, schedule_time, complete_time FROM item`); err != nil {
```

Comment:
We should filter by items that have not been deleted `WHERE delete_time IS NULL`.